src/components/Book.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Book.css';
+
+function Book({ book, deleteBook }) {
+  return (
+    <div>
+      <li>
+        <div>
+          <h3>title:</h3>
+          {book.title}
+        </div>
+        <div>
+          <p>author:</p>
+          {book.author}
+        </div>
+        <div>
+          <p>category:</p>
+          {book.category}
+        </div>
+        <button type="button" onClick={() => deleteBook(book.id)}>
+          Delete
+        </button>
```

Comment:
- [ ] Great job so far @efmomoh There is one change I will suggest you implement and you'll be on your way to the next Milestone 👍 
- [ ] Currently, I am unable to delete a book and this is because you are not making use of the `removeBook` method that was created in the `bookslice.js` file. So kindly export that method and use it here to make it possible for added books to be deleted 😉